### PR TITLE
Change bulkUpserter retry period and prevent erroring silently

### PR DESCRIPTION
**Pull request by sebwittr**
# Purpose

_In 2-3 sentences - what does this code actually do, and why?_

Our bulkIndex was failing silently, so now it will catch and log the errors. We also slightly increased the retry period in case it needed more time. 

# Tickets

_What Trello tickets (if any) are associated with this PR?_

# Contributors

@sebwittr 

Use the **"Assignees"** feature in Github

# Feature List

_Expand on the purpose - what does this code do, and how? This should be a list of the changes, more in-depth and technical_

- Increased retry period to 1000 to bulkIndex
- Added a try/catch to see if it failed, and logged it
-

# Notes (Optional)

_Is there anything reviewers should note? Do we need to add something later? Any followups? Was this ticket easier/harder than predicted?_

# Reviewers

**Primary**:

@hankewyczz  @Lucas-Dunker

### CODE REVIEW

File: utils/elastic.ts [hunk 1]
```diff
@@ -257,13 +257,19 @@ export class Elastic {
...
           bulk.push(map[id]);
         }
-        // assumes that we are writing to the ES index name, not the ES alias (which doesn't have write privileges)
-        const res = await this.retryBulkQuery(indexName, bulk);
-
-        macros.log(
-          `indexed ${chunkNum * BULKSIZE + chunk.length} docs into ${indexName}`
-        );
-        return res;
+        try {
```
Comment (hankewyczz) [inline utils/elastic.ts:260]: We should toss this functionality into `retryBulkQuery`

**Review by hankewyczz — CHANGES_REQUESTED** [review]

**Review by Lucas-Dunker — APPROVED** [review]
LGTM 🙏

File: utils/elastic.ts (hunk 1 above)
Comment (sebwittr) [reply utils/elastic.ts:260]: retryBulkQuery already has some try catch functionality, and bulkIndexFromMap was the one that was silently erroring.

File: utils/elastic.ts [hunk 2]
```diff
@@ -318,6 +319,11 @@ export class Elastic {
         return await client.bulk({ index: indexName, body: bulk });
       } catch (e) {
         macros.log(`Caught while bulk upserting: ${e.name} - ${e.message}`);
+
+        if (i == MAX_RETRY_ATTEMPTS - 1) {
```
Comment (hankewyczz) [inline utils/elastic.ts:323]:
```suggestion
        if (i === MAX_RETRY_ATTEMPTS - 1) {
```

**Review by hankewyczz — APPROVED** [review]

**Pull request merged**